core/lib/generators/refinery/engine/templates/Gemfile

```diff
@@ -25,6 +25,7 @@ platforms :ruby do
 end
 
 group :development, :test do
+  gem 'rspec-its' # for the model's validation tests.
```

Comment:
Does Refinery itself need it? I feel like people should know they're only opting as their extensions are using it.